test/ui-test/testSuites/suite_status/tst_groupChat/test.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Status Desktop Group Chat
+
+	As a user I want to use group chat functionality.
+
+  	Scenario: As an admin user I want to create a group chat with my contacts and invited users can send messages
+
+  	   Given the user starts the application with a specific data folder ../../../testSuites/suite_status/tst_groupChat/data
+  	    When the user tester123 logs in with password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When the user creates a group chat adding users
+        	 | Athletic |
+        	 | Nervous  |
+		Then the group chat is created
+		 And the group chat history contains created the group message
+		 And the group chat title is Athletic&Nervous
+		 And the group chat contains the following members
+        	 | Athletic |
+        	 | Nervous  |
+		 And the group chat is up to chat sending Admin user message sent!! message
```

Comment:
I don't understand this sentence?

Reply:
I think it means that the last messages is X. 
But I agree, this is hard to understand.
Maybe we can replace it by something like "And the last message of the group chat is |any|"

Reply:
Correct.. I will update it including the "". We can agree on that.. if we use long strings we can add "" as part of the step definition.